<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coinbaseAccountSetupWindow</class>
 <widget class="QDialog" name="coinbaseAccountSetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coinbase Accounts</string>
  </property>
  <widget class="QListWidget" name="mAccountList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mDoneButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mAddButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mRemoveButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
